Form/Type/MediaType.php

```diff
@@ -77,14 +76,34 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     /**
      * {@inheritdoc}
+     *
+     * @deprecated To be removed when bumping to Symfony 2.7+, use MediaType::configureOptions() instead
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
+        if ($resolver instanceof OptionsResolver) {
```

Comment:
In this case, `MyOwnOptionsResolver` should inherit `OptionsResolver` on your project.

And I don't think this can be apply and this project.

I don't know more. Maybe @stof could give a better review/explanation than me.

cc @rande